internal/provisioner/controller/gateway_test.go

```diff
@@ -1015,7 +1015,8 @@ func TestGatewayReconcile(t *testing.T) {
 				Spec: contourv1alpha1.ContourDeploymentSpec{
 					Envoy: &contourv1alpha1.EnvoySettings{
 						NetworkPublishing: &contourv1alpha1.NetworkPublishing{
-							Type: contourv1alpha1.ClusterIPServicePublishingType,
+							Type:                  contourv1alpha1.ClusterIPServicePublishingType,
```

Comment:
Change this to a NodePort service since it's not valid to set an ExternalTrafficPolicy on a ClusterIP service:
```suggestion
							Type:                  contourv1alpha1.NodePortServicePublishingType,
```